build/targets/KoreBuild.DefaultLifecycle.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+
+<!--
+###################################################################
+Standard lifecycle targets.
+
+When extending the solution build, chain off one of these.
+
+The 'DependsOnTargets' are defined in KoreBuild.Common.targets
+###################################################################
+-->
+  <Target Name="PrepareForBuild" DependsOnTargets="CleanArtifacts;CleanSolutions" />
+
+  <Target Name="Initialize" DependsOnTargets="RestoreSolutions" />
```

Comment:
Intentionally removed. As far as I can tell, no one uses grunt, we never supported gulp, and SignalR is the only one that actually builds JavaScript...and it doesn't rely on KoreBuild to do that. The other usages of JS stuff are for samples.

cref #173